mermin/src/metrics/registry.rs

```diff
@@ -17,87 +17,108 @@ lazy_static! {
+            .namespace("mermin"),
+        &["type"]  // received, dropped, filtered
+    ).expect("failed to create ringbuf_packets metric");
+
+    pub static ref USERSPACE_RINGBUF_BYTES: IntCounter = IntCounter::with_opts(
+        Opts::new("ringbuf_bytes_total", "Total number of bytes received from the userspace ring buffer")
+            .namespace("mermin")
+    ).expect("failed to create ringbuf_bytes metric");
+
+    pub static ref USERSPACE_CHANNEL_CAPACITY: IntGaugeVec = IntGaugeVec::new(
+        Opts::new("channel_capacity", "Capacity of internal channels")
+            .namespace("mermin"),
+        &["channel"]  // packet_worker, exporter
+    ).expect("failed to create channel_capacity metric");
+
+    pub static ref USERSPACE_CHANNEL_SIZE: IntGaugeVec = IntGaugeVec::new(
+        Opts::new("channel_size", "Current number of items in channels")
+            .namespace("mermin"),
+        &["channel"]  // packet_worker, exporter
+    ).expect("failed to create channel_size metric");
+
+    pub static ref USERSPACE_CHANNEL_SENDS: IntCounterVec = IntCounterVec::new(
+        Opts::new("channel_sends_total", "Total number of channel send operations")
+            .namespace("mermin"),
+        &["channel", "status"]  // status: success, error; channel: packet_worker, exporter
+    ).expect("failed to create channel_sends metric");
+
+    // ============================================================================
+    // Flow Span Lifecycle Metrics (subsystem: span)
     // ============================================================================
 
-    /// Total number of flows created.
     pub static ref FLOWS_CREATED: IntCounterVec = IntCounterVec::new(
-        Opts::new("mermin_flows_created_total", "Total number of flows created")
+        Opts::new("span_flows_created_total", "Total number of flows created")
             .namespace("mermin"),
         &["interface"]
     ).expect("failed to create flows_created metric");
 
-    /// Total number of flows expired/removed.
     pub static ref FLOWS_EXPIRED: IntCounterVec = IntCounterVec::new(
-        Opts::new("mermin_flows_expired_total", "Total number of flows expired")
+        Opts::new("span_flows_expired_total", "Total number of flows expired")
             .namespace("mermin"),
         &["reason"]  // timeout, recorded, error, guard_cleanup
     ).expect("failed to create flows_expired metric");
 
-    /// Current number of active flows being tracked.
     pub static ref FLOWS_ACTIVE: IntGaugeVec = IntGaugeVec::new(
-        Opts::new("mermin_flows_active", "Current number of active flows")
+        Opts::new("span_flows_active", "Current number of active flows")
             .namespace("mermin"),
         &["interface"]
     ).expect("failed to create flows_active metric");
 
-    /// Histogram of flow durations.
     pub static ref FLOW_DURATION: Histogram = Histogram::with_opts(
```

Comment:
This doesn't feel like a metric... I don't know in what use-case I would care to see this.